### PR TITLE
Issue 66 add story page component

### PR DESCRIPTION
**Pull request by mhtien**
## Goal

Create Add Story Page Component

## Todos:
- [ ] Outstanding todo
- [x] Completed todo
 
## Implementation Decisions

### CODE REVIEW

File: src/components/AddStory.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  
+
+  return (
+      <main>
+        <Box display="flex" justifyContent="space-between" mt="1rem">
+          <h1>Add Story</h1>
+          <Button variant="outlined">Cancel</Button>
+        </Box>
+        <form>
+        <TextField className="url-input"
```
Comment (kkyeboah) [inline src/components/AddStory.tsx:28]: Could you also make the text in this text field a placeholder so you don't have to delete the text before adding your own URL?

**Review by kkyeboah — CHANGES_REQUESTED** [review]
This looks great but we have a few comments:

Could you please make sure that your PR only contains changes for your component?

**Comment by mhtien** [comment]
Hi @kkyeboah,

Comments noted, I opened up another pull request #76 that just includes the component, and the input field has a placeholder now.

Let me know if you have any more comments thanks!

**Comment by jpetto** [comment]
closing in favor of #76

**Pull request closed without merging**